types/common/evaluable.d.ts

```diff
@@ -1,36 +1,36 @@
-/**

```

Comment:
What is the different, tab vs spaces? I wonder if this is na PC vs MAC setup thing

Reply:
I think the files currently in the repository have windows line endings 
<img width="854" alt="Screen Shot 2020-08-06 at 11 54 28 AM" src="https://user-images.githubusercontent.com/44178530/89553506-9bce0000-d7db-11ea-8e8e-9cc33ed028d4.png">

Reply:
hmm, will check on my end than